lib/l10n_utils/templatetags/fluent.py

```diff
@@ -2,17 +2,36 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import bleach
 import jinja2
 from django_jinja import library
 from markupsafe import Markup
 
 from lib.l10n_utils import fluent
 
+TAGS_ALLOWED_IN_FLUENT_STRINGS = {
```

Comment:
I see some uses of `<i>` in our l10n files. This should be safe to add i think

Reply:
Interesting - I don't see them with my grepping. Got an example filepath?

Reply:
I just did a quick search in VSCode:

<img width="493" alt="image" src="https://user-images.githubusercontent.com/400117/218105179-9d8dfd6c-87e1-4b93-a9ef-9c76d4337b34.png">